### ISSUE TITLE
why training accuracy is very small ?

### ISSUE THREAD

**Issue by century3** [loewenm/deep-learning-and-the-game-of-go#1]
after running sample train_generator.py of chapter 7, why my accuracy is so small?:
C:\users\cham wang\PycharmProjects\deep_learning_and_the_game_of_go-master\code\examples\train_generator.py:49: UserWarning: `Model.fit_generator` is deprecated and will be removed in a future version. Please use `Model.fit`, which supports generators.
  model.fit_generator(generator=generator.generate(batch_size, num_classes),  # <1>
Epoch 1/5
2023-01-27 15:06:20.358386: I tensorflow/stream_executor/cuda/cuda_dnn.cc:384] Loaded cuDNN version 8700
2023-01-27 15:06:21.624890: I tensorflow/stream_executor/cuda/cuda_blas.cc:1614] TensorFloat-32 will be used for the matrix multiplication. This will only be logged once.
88/88 [==============================] - 4s 19ms/step - loss: 5.8891 - accuracy: 0.0023 - val_loss: 5.8884 - val_accuracy: 0.0033
Epoch 2/5
88/88 [==============================] - 1s 17ms/step - loss: 5.8882 - accuracy: 0.0030 - val_loss: 5.8875 - val_accuracy: 0.0038
Epoch 3/5
88/88 [==============================] - 1s 17ms/step - loss: 5.8873 - accuracy: 0.0031 - val_loss: 5.8866 - val_accuracy: 0.0034
Epoch 4/5
88/88 [==============================] - 2s 18ms/step - loss: 5.8864 - accuracy: 0.0033 - val_loss: 5.8857 - val_accuracy: 0.0036
Epoch 5/5
88/88 [==============================] - 2s 17ms/step - loss: 5.8854 - accuracy: 0.0032 - val_loss: 5.8845 - val_accuracy: 0.0036
C:\users\cham wang\PycharmProjects\deep_learning_and_the_game_of_go-master\code\examples\train_generator.py:56: UserWarning: `Model.evaluate_generator` is deprecated and will be removed in a future version. Please use `Model.evaluate`, which supports generators.
  model.evaluate_generator(generator=test_generator.generate(batch_size, num_classes),
